dev/reference/tools/loader.md

```diff
@@ -97,13 +97,10 @@ user = "root"
 password = ""
 port = 4000
 
-# `sql_mode` used when loading data. If the `sql-mode` value is not provided or set to "@DownstreamDefault", downstream global `sql_mode` will be used
+# `sql_mode` of session level used to connect to the database when loading data. If `sql-mode` is not provided or set to "@DownstreamDefault", the global `sql_mode` for downstream will be used.
 # sql-mode = ""
-
-# max-allowed-packet sets `max_allowed_packet` for database connection.
-# `max-allowed-packet=0` can be used to automatically fetch the
-# `max_allowed_packet` variable from server on every connection.
-max-allowed-packet = 67108864
+# `max_allowed_packet` sets the maximum data packet allowed for database connection, which corresponds to the `max-allowed-packet` in system parameters. If set to 0, the global `max-allowed-packet` for downstream will be used.
+max_allowed_packet = 67108864
```

Comment:
oh this is misused, `max-allowed-packet` is used in this config template, and `max_allowed_packet` is the system parameter in the database